src/AVFoundation/AVTypes.cs

```diff
@@ -363,20 +451,76 @@ public struct AVSampleCursorDependencyInfo {
 
 		[MarshalAs (UnmanagedType.I1)]
 		public bool HasRedundantCoding;
+#endif
 	}
 
+#if !(XAMCORE_5_0 || __IOS__ || __TVOS__)
+	[StructLayout (LayoutKind.Sequential)]
+#if COREBUILD
+	public
+#endif
+	struct AVSampleCursorDependencyInfo_Blittable {
+		byte indicatesWhetherItHasDependentSamples;
+		byte hasDependentSamples;
+		byte indicatesWhetherItDependsOnOthers;
+		byte dependsOnOthers;
+		byte indicatesWhetherItHasRedundantCoding;
+		byte hasRedundantCoding;
+
+		public bool IndicatesWhetherItHasDependentSamples {
+			get => indicatesWhetherItHasDependentSamples != 0;
+			set => indicatesWhetherItHasDependentSamples = value.AsByte ();
+		}
+
+		public bool HasDependentSamples {
+			get => hasDependentSamples != 0;
+			set => hasDependentSamples = value.AsByte ();
+		}
+
+		public bool IndicatesWhetherItDependsOnOthers {
+			get => indicatesWhetherItDependsOnOthers != 0;
+			set => indicatesWhetherItDependsOnOthers = value.AsByte ();
+		}
+
+		public bool DependsOnOthers {
+			get => dependsOnOthers != 0;
+			set => dependsOnOthers = value.AsByte ();
+		}
+
+		public bool IndicatesWhetherItHasRedundantCoding {
+			get => indicatesWhetherItHasRedundantCoding != 0;
+			set => indicatesWhetherItHasRedundantCoding = value.AsByte ();
+		}
+
+		public bool HasRedundantCoding {
+			get => hasRedundantCoding != 0;
+			set => hasRedundantCoding = value.AsByte ();
+		}
+
+		public AVSampleCursorDependencyInfo ToAVSampleCursorDependencyInfo ()
+		{
+			var rv = new AVSampleCursorDependencyInfo ();
+			rv.IndicatesWhetherItHasDependentSamples = IndicatesWhetherItHasDependentSamples;
+			rv.HasDependentSamples = HasDependentSamples;
+			rv.IndicatesWhetherItDependsOnOthers = IndicatesWhetherItDependsOnOthers;
+			rv.DependsOnOthers = DependsOnOthers;
+			rv.IndicatesWhetherItHasRedundantCoding = IndicatesWhetherItHasRedundantCoding;
+			rv.HasRedundantCoding = HasRedundantCoding;
+			return rv;
+		}
```

Comment:
Reminder we can do something like this in the new c#
```suggestion
		public AVSampleCursorDependencyInfo ToAVSampleCursorDependencyInfo ()
		{
			var rv = new AVSampleCursorDependencyInfo () {
					IndicatesWhetherItHasDependentSamples = IndicatesWhetherItHasDependentSamples,
					HasDependentSamples = HasDependentSamples,
					IndicatesWhetherItDependsOnOthers = IndicatesWhetherItDependsOnOthers,
					DependsOnOthers = DependsOnOthers,
					IndicatesWhetherItHasRedundantCoding = IndicatesWhetherItHasRedundantCoding,
					HasRedundantCoding = HasRedundantCoding,
			}
			return rv;
		}
```
Do I care we did not use it, no :)